chickadee/processes/wps_CI.py

```diff
@@ -0,0 +1,137 @@
+from pywps import Process, ComplexInput, ComplexOutput, LiteralInput, FORMATS
+from pywps.app.Common import Metadata
+
+from wps_tools.utils import log_handler
+from wps_tools.io import log_level
+from chickadee.utils import logger, get_package
+
+
+class CI(Process):
+    def __init__(self):
+        self.status_percentage_steps = {
+            "start": 0,
+            "process": 10,
+            "build_output": 95,
+            "complete": 100,
+        }
+        inputs = [
+            ComplexInput(
+                "gcm_file",
+                "GCM NetCDF file",
+                abstract="Filename of GCM simulations",
+                min_occurs=1,
+                max_occurs=1,
+                supported_formats=[FORMATS.NETCDF, FORMATS.DODS],
+            ),
+            ComplexInput(
+                "obs_file",
+                "Observations NetCDF file",
+                abstract="Filename of high-res gridded historical observations",
+                min_occurs=1,
+                max_occurs=1,
+                supported_formats=[FORMATS.NETCDF, FORMATS.DODS],
+            ),
+            LiteralInput(
+                "varname",
+                "Variable to Downscale",
+                abstract="Name of the NetCDF variable to downscale (e.g. 'tasmax')",
+                min_occurs=1,
+                max_occurs=1,
+                data_type="string",
+            ),
+            LiteralInput(
+                "out_file",
+                "Output NetCDF File",
+                abstract="Filename to create with the climate imprint outputs",
+                min_occurs=0,
+                max_occurs=1,
+                data_type="string",
+            ),
+            log_level,
```

Comment:
These inputs are shared for several of the processes, it would save code in all of the processes to move them into a `utils` or `io` module.